core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -103,7 +103,7 @@
         public enum CommandState {
             /** Request (command line) is in preparation to be generated and sent, but not yet sent to server. */
             REQUEST_IN_PREPARATION,
-            /** Request (command line) is confirmed sent to the the server. */
+            /** Request (command line) is confirmed sent to the server. */
```

Comment:
Good. But this change is not needed in this PR. Avoid unrelated changes and create separate pull requests that can be merged quickly for the typo/warning/formatting only issues